apps/src/gamelab/textConsoleModule.js

```diff
@@ -0,0 +1,33 @@
+import {CLEAR_CONSOLE, ADD_MESSAGE} from './actions';
+
+export default function textConsole(state, action) {
+  state = state || [];
+  switch (action.type) {
+    case CLEAR_CONSOLE:
+      return [];
+    case ADD_MESSAGE:
+      return [
+        ...state.slice(0, 999),
```

Comment:
We need to limit it by some amount, and 1000 seemed like within most normal use-cases. Also, since the display is limited to 6 lines, 1000 would be a lot. Would you suggest a different amount?

Reply:
would you add a comment to indicate that this limit is arbitrary, and could easily be changed in the future?

Reply:
To be 100% sure, it's a separate state object because you used `combineReducers` to namespace them?

Reply:
After a bit of research, yes. combineReducers gives that functionality and it looks like combineReducers is called on gamelab/reducers.
I confirmed by hooking it up to a debugger that the only state this has access to is its own.